provision.sh

```diff
@@ -177,24 +177,28 @@ if [ ! $SKIP_REQUIREMENTS ] ; then
       exit 1
   else
     # Install virtualenv
-    which -a virtualenv >> /dev/null
+    which -a pipenv >> /dev/null
     if [[ $? != 0 ]] ; then
-      pip install virtualenv
+      pip install pipenv
     fi
+    cd $ROOT/ansible
+    VENV=`pipenv --venv`
```

Comment:
@tharna Is this used only for the `source $VENV/bin/activate` command later in the code? If it is, I don't see any need for this.